src/main/java/com/expensetrackersvc/controller/api/CategoryController.java

```diff
@@ -0,0 +1,37 @@
+package com.expensetrackersvc.controller.api;
+
+import com.expensetrackersvc.model.Category;
+import com.expensetrackersvc.repository.CategoryRepository;
+import io.swagger.v3.oas.annotations.Operation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/category")
+public class CategoryController {
+
+    @Autowired
+    CategoryRepository categoryRepository;
+
+
+    @Operation(summary = "create expense")
+    @PostMapping("/addCategory")
```

Comment:
We can directly use POST /v1/category call to create category. Don't need to add extra '/addCategory' URL. Just a suggestion